python/revng/api/_capi.py

```diff
@@ -0,0 +1,44 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+from pathlib import Path
+
+from cffi import FFI
+from cffi.backend_ctypes import CTypesBackend
+
+from .utils import make_c_string
+
+ORCHESTRA_ROOT = Path(__file__).parent.parent.parent.parent.parent
+HEADERS_DIR = ORCHESTRA_ROOT / "include" / "revng" / "PipelineC"
+
+header_paths = [
+    HEADERS_DIR / "c_typedefs.h",
+    HEADERS_DIR / "prototypes.h",
+]
+
+ffi = FFI()
+for header_path in header_paths:
+    with open(header_path) as header_file:
+        lines = []
+        for line in header_file:
+            if not line.startswith("#"):
+                lines.append(line)
+        ffi.cdef("\n".join(lines))
+
+LIB_DIR = ORCHESTRA_ROOT / "lib"
+LIBRARY_PATH = str(LIB_DIR / "librevngPipelineC.so")
+
+ARGV = []
+ANALYSIS_LIBRARIES = [
+    str(LIB_DIR / "revng/analyses/librevngFunctionIsolation.so"),
+]
```

Comment:
We can't hardcode this. You need to integrate with `collect_files` from `python/revng/cli/support.py`.
Also, this should be in a `main` with a `--prefix` argument. If this is not possible we need to find a similar solution (env var?).

Reply:
After squashing there is an env var present and a more generic `.so` handling, if need be it can be changed accordingly